source/jsonnet/england-wales/ccs/blocks/individual/another_uk_address.jsonnet

```diff
@@ -24,7 +38,7 @@ local question(title) = {
         {
           label: 'Previous home',
           value: 'Previous home',
-          description: 'Moved since 13 October 2019',
+          description: description,
```

Comment:
I'm not sure why we keep creating variables for small blocks that aren't being re-used anywhere. Here it would be clearer if the description Jsonnet was inline.